paket.dependencies

```diff
@@ -31,4 +31,5 @@ nuget FsCheck.Xunit
 nuget Machine.Specifications.Should
 nuget Machine.Specifications.Runner.Console
 nuget Nancy.Testing
-nuget xunit.extensions
\ No newline at end of file
+nuget xunit.extensions
+nuget Newtonsoft.Json
```

Comment:
Hey, we asked whether you wanted a different serializer up front! Keeping Json.Net makes sense, though, given that it's included elsewhere. =)

I'm actually able to build from both mac and windows without any additional modifications. This might be a side effect of NewtonSoft.Json being included in the Deploy and Deploy.Lib projects... Or I might be totally misunderstanding your comment!